src/delimiter.js

```diff
@@ -318,6 +313,112 @@ const makeStackedDelim = function(delim, heightTotal, center, options, mode,
         Style.TEXT, options, classes);
 };
 
+const sqrtInnerSVG = {
+    main : `<svg viewBox=\'0 0 400000 1000\' preserveAspectRatio=\'xMinYMin
```

Comment:
Re: template strings. Yes, indeed. And that's true for the comments below, as well.